src/js/button.js

```diff
@@ -3,10 +3,7 @@
  */
 import ClickableComponent from './clickable-component.js';
 import Component from './component';
-import * as Events from './utils/events.js';
```

Comment:
On the topic of using the `import 'foo';` style, I don't think it puts the exports into local scope. I think it simply runs the module code. MDN says:

> Import an entire module for side effects only, without importing any bindings.